### ISSUE TITLE
Feature Request: Add materials / systhesis indicators

### ISSUE THREAD

**Issue by mpfj** [EDDiscovery/EDDiscovery#873]
I would be nice if, as you scan a system, it checks any of the materials to see if you could use them in Jumponium and/or ARM Refill synthesis.

For Jumponium:
J1 (Basic), flag if both Vanadium and Germanium had been found.
J2 (Standard), flag if Vanadium, Germanium, Cadmium and Niobium are all present.
J3 (Premium), flag if Arsenic, Niobium, Yttrium and Polonium are present.

For AFM Refills:
Basic : Nickel, Zinc, Chromium, Vanadium
Standard : Tin, Manganese, Vanadium, Molybdenum and Zirconium
Premium : Zinc, Chromium, Vanadium, Zirconium, Tellurium and Ruthenium

**Comment by RedfoxCh** [comment]
I guess this could be considered as related to #702.

**Comment by robbyxp1** [comment]
and #906

**Comment by robbyxp1** [comment]
Closed due to 906.. new synth panel

**Comment by mpfj** [comment]
The synthesis panel is good, but I was hoping for a slightly different result. 

As you scan planets in a system, it'll gradually show you what can be created using the materials  you've found just in that system. 

So you can then tell what can be synthesised *just* in that system. 

Hope I've explained that okay!!

**Comment by RedfoxCh** [comment]
@mpfj Interesting. Is that really your process? You synthesise what you *can*? Personally, I usually know what I *want* to synthesise, then need to know what materials are needed to do that, and finally need to know which of those I'm still missing.

A luxury option could be EDD telling us: "Hey CMDR, I see that you're still missing 6 Yttrium for those L3 FSD boosts. FYI this systems has a planet with high Yttrium deposits."

**Comment by mpfj** [comment]
Yes and no.

As I scan planets, it's nice to see what I *can* synthesize using what's already scanned, if I needed to.  But that doesn’t mean I do actually synthesize those things.

Of course, I could look at the list of mats each planet has and work it out, but I'd have to remember what all the ingredients are.  That's what computers are for ...

But, for me, it'd be a quick indicator that somewhere in the planets I've scanned, there's all the mats needed for making xyz product.

That way I can quickly see if it's worth stocking up on mats (especially if they're all on a single planet !!)
